src/sidebar/services/annotations-exporter.ts

```diff
@@ -158,6 +158,64 @@ export class AnnotationsExporter {
     return `${headers}\n${annotationsContent}`;
   }
 
+  buildHTMLExportContent(
+    annotations: APIAnnotationData[],
+    {
+      groupName = '',
+      displayNamesEnabled = false,
+      defaultAuthority = '',
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: ExportOptions = {},
+  ): string {
+    const { uri, title, uniqueUsers, replies, extractUsername } =
+      this._exportCommon(annotations, {
+        displayNamesEnabled,
+        defaultAuthority,
+      });
+
+    const htmlAnnotations = annotations.map((annotation, index) => {
+      const page = pageLabel(annotation);
+      const lines = [
+        `<p>${annotation.created}</p>`,
+        `<p>Comment: ${escapeHtml(annotation.text)}</p>`,
+        `<p>${escapeHtml(extractUsername(annotation))}</p>`,
+        `<p>Quote: <blockquote>${escapeHtml(
+          quote(annotation),
+        )}</blockquote></p>`,
+        annotation.tags.length > 0
+          ? `<p>Tags: ${annotation.tags.map(escapeHtml).join(', ')}</p>`
+          : undefined,
+        page ? `<p>Page: ${escapeHtml(page)}</p>` : undefined,
+      ].filter(Boolean);
+
+      return trimAndDedent`
+        <article>
+          <strong>Annotation ${index + 1}:</strong>
+          ${lines.join('\n')}
+        </article>`;
+    });
+
+    return trimAndDedent`
+      <header>
+        <p><time datetime="${now.toISOString()}">${now.toISOString()}</time></p>
+        <p><strong>${escapeHtml(title)}</strong></p>
```

Comment:
Rather than doing the escaping manually for every element, you can move that into the template string, you can move that into the tag function for the template string.

Something we could alternatively consider using is https://github.com/preactjs/preact-render-to-string, since it runs in the browser. This would mean we write the HTML the same way we author UI components, and render it using this library.

Reply:
> Rather than doing the escaping manually for every element, you can move that into the template string, you can move that into the tag function for the template string.

We can't do that, because it would then escape our own hardcoded tags that we want to preserve. Unless I'm not understanding what you mean.

> Something we could alternatively consider using is https://github.com/preactjs/preact-render-to-string, since it runs in the browser. This would mean we write the HTML the same way we author UI components, and render it using this library.

I like this option, but I was trying to keep the output as human-friendly as possible. If it allows it, I think it's not a bad option at all.

Reply:
> > Rather than doing the escaping manually for every element, you can move that into the template string, you can move that into the tag function for the template string.
> 
> We can't do that, because it would then escape our own hardcoded tags that we want to preserve. Unless I'm not understanding what you mean.

Oh, I was in fact not understanding what you meant. I just got it 😅. Yeah, it makes sense.